src/coinjoin/client.cpp

```diff
@@ -47,6 +48,11 @@ PeerMsgRet CCoinJoinClientQueueManager::ProcessDSQueue(const CNode& peer, CDataS
     CCoinJoinQueue dsq;
     vRecv >> dsq;
 
+    {
+        LOCK(cs_main);
+        Assert(peerman)->EraseObjectRequest(peer.GetId(), CInv(MSG_DSQ, dsq.GetHash()));
```

Comment:
What if by any reason  `CCoinJoinClientQueueManager` will be initialized before `peerman`?

Should we just change assert here to if?

```suggestion
        if (peerman) peerman->EraseObjectRequest(peer.GetId(), CInv(MSG_DSQ, dsq.GetHash()));
```